source/src/cip/cipcommon.c

```diff
@@ -1264,6 +1289,146 @@ int DecodePaddedEPath(CipEpath *epath,
   return number_of_decoded_elements * 2 + 1; /* number_of_decoded_elements times 2 as every encoding uses 2 bytes */
 }
 
+EipStatus Create(CipInstance *RESTRICT const instance,
```

Comment:
Create is probably a too generic name, please change to CipCreateService (or similar)